lib/Sema/TypeCheckDecl.cpp

```diff
@@ -3173,7 +3173,7 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
         TAD->getUnderlyingTypeLoc().setInvalidType(TC.Context);
       } else if (TAD->getDeclContext()->isGenericContext()) {
         TAD->setInterfaceType(
-          TC.getInterfaceTypeFromInternalType(TAD->getDeclContext(),
+        TC.getInterfaceTypeFromInternalType(TAD->getDeclContext(),
                                               TAD->getType()));
```

Comment:
I would go as far as saying that white space had meaning and now is gone. It was enforcing that getInterfaceTypeFromInternalType is an argument passed to setInterfaceType. 
It's merged now but would you oppose adding this space back in? I'm not one for PR (just to correct this) with such minimal changes but...